foundation.yaml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: teaspoons
+  description: |
+    Terra Scientific Pipelines Service, or teaspoons, facilitates running a number of defined scientific pipelines 
+    on behalf of users that users can't run themselves in Terra. The most common reason for this is that the pipeline
+    accesses proprietary data that users are not allowed to access directly, but that may be used as e.g. a reference panel
+    for imputation.
+  tags:
+    - java
+    - dsp
+    - terra
+    - springboot
+  annotations:
+    github.com/project-slug: databiosphere/terra-scientific-pipelines-service
+  links:
+    - url: https://grafana.dsp-devops.broadinstitute.org/d/3VhQ1_0Sz/teaspoons?orgId=1
+      title: Teaspoons Grafana Dashboard
+      icon: dashboard
+spec:
+  type: service
+  lifecycle: experimental
+  owner: teaspoons
+  system: terra
+  dependsOn:
+    - component:sam
+    - component:terra-workspace-manager
```

Comment:
i think we depend on more currently

Reply:
yeah, though wsm is the only other one that's actually in backstage right now. (sam was in the bmp template so i kept it but there's not a sam backstage component.) not sure whether to add them all even if they don't exist

Reply:
yea if we have sam and they dont have a backstage component then i would add the rest too